mex-wholebodymodel/matlab/wrappers/+WBM/+utilities/eul2rotm.m

```diff
@@ -0,0 +1,64 @@
+function rotm = eul2rotm(eul, sequence)
```

Comment:
There are a lot of this utility functions, did you write them or you copied them form somewhere else? In the latter case it is important to know the source to avoid copyright infrangment problems in future.